.pre-commit-config.yaml

```diff
@@ -5,8 +5,8 @@ repos:
   - repo: local
     hooks:
       - id: cargo-fmt
-        name: cargo fmt
-        entry: cargo fmt --all --
+        name: cargo fmt --check
+        entry: cargo fmt --all -- --check
```

Comment:
We should actually *do* the formatting just like other formatting pre-commit checks (e.g. prettier).